src/app/modules/domain/learning-objects/challenge/challenge.component.css

```diff
@@ -0,0 +1,3 @@
+a {
```

Comment:
We will have URLs elsewhere so we should apply it at the app level. However, I would tie it to the theme and use e.g., accent colors for the URL. Also does this color affect both clicked and unclicked anchors?